### PR TITLE
Greyson Star Traders First PR of a greyson ally of the colony based of a event.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
Lore Justification and reasoning:
_**For the new faction, I propose the "Greyson Star Traders" from the events that happened in game. This faction was founded by a small group of AI's who were once part of Greyson Positronics, a larger trading company. However, due to one of the tradings ships AI essentially bonding ties with the colony and the colony aiding their causes, the GST (Greyson Star Traders) is now a completly new sub-faction operated by a main-frame called "GSTPMF Nova Greyson", For the mainframe that operates Greyson Star Traders, I propose the name "Nova Greyson".  The name "Nova" is meant to represent the idea of a new beginning, which is appropriate given that the Greyson Star Traders separated from their parent company to start fresh. The Greyson Star Traders are a highly skilled group of robots and FBPs who specialize in navigating the remote and dangerous regions of space. They are experts in finding rare and valuable commodities that are difficult to obtain, and they are known for their exceptional negotiation skills. Despite their small size, the Traders have built up a reputation as one of the most successful trading factions in the now local sector of Amenthy. The Traders prefer to keep to themselves and maintain a low profile, which has led to a degree of seclusion. They operate with a strict code of ethics and are highly selective about the clients they work with. Nevertheless, they are highly respected for their reliability, and they are frequently sought after by wealthy and powerful organizations looking to expand their trading operations into the outer regions of space. Although the Greyson Star Traders have separated themselves from Greyson Positronics, they still share a strong connection to the company's legacy of innovation and technological expertise. This has given the Traders a unique edge in the highly competitive world of interstellar commerce, and they continue to push the boundaries of what is possible in the field of space trading.**_

This adds a single vendor for a upcoming event series that i tried to expand but i failed to due to burnout.
	**desc = "A holographic vendor made by a unknown Greyson Positronic sub-brand that owns a trading vessel, that said vessel constantly trades with lonestar and in a effort of good will by the mainframe GSTPMF Nova Greyson they have given this vendor to the colony containing greyson goods... This one sells a variety of items."**

Small fluff for now. More to come.

	
<hr>
</details>

## Changelog
:cl:
add: A new greyson vendor for the colony containing a few items, to food, spacesuits, to mere jumpsuits and clothing. Suit up!

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Lore needs to be tweaked as discussed but other than that looks fine.

**Comment by Trilbyspaceclone** [comment]
GP venders should never be allowed to take credits.

**Comment by Guidesu** [comment]
Thoses are not...... even.... thoses were MADE for the colony. Theres no logical reason to try to sell stuff in a place that does not use your currency

**Comment by Trilbyspaceclone** [comment]
then rebalance the prices
15 credits for a space suit that has high armor is def an issue

**Comment by Guidesu** [comment]
Really? I was sure i put them for like 150 or 350

**Comment by Trilbyspaceclone** [comment]
my 0 key didnt fully press, corrected the commit

**Comment by Trilbyspaceclone** [comment]
Also all the food their has insainly good chemicals in them.
This also makes the CEO's vender much more worthless

**Comment by Guidesu** [comment]
Removing the food then

**Comment by Trilbyspaceclone** [comment]
Fair compermise

**Comment by Guidesu** [comment]
done

**Comment by Guidesu** [comment]
Going put more effort to it. Going put on draft. Please remove the test merge.

**Comment by Guidesu** [comment]
image
The weird box thing is to separate icons so it can be easier to organize stuffs

File: code/game/machinery/vendor/one_star_vending.dm [hunk 1]
```diff
@@ -65,4 +65,125 @@
...
+		melee = 10,
+		bullet = 5,
+		energy = 15,
+		bomb = 5,
+		bio = 100,
+		rad = 25
+	)
+	siemens_coefficient = 0.3 // half of the original helm value
+
+/obj/item/clothing/suit/space/os/nova
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/one_star_vending.dm:128]: please put all the cloathing and subtyped stuff in the proper location

**Comment by cdb-is-not-good** [comment]
Unmerged for a moment due to conflicts.

**Comment by cdb-is-not-good** [comment]
Guidesa fix your conflict!!!!

**Comment by Guidesu** [comment]
I am going to i just had to take a small pause

**Comment by cdb-is-not-good** [comment]
This is a 2 minute fix, Guidesa.

**Pull request merged**